code/modules/mob/living/carbon/human/species_types/IPC.dm

```diff
@@ -0,0 +1,177 @@
+/datum/species/ipc // Beestation's IPCs
+	name = "IPC" //inherited from the real species, for health scanners and things
+	id = "ipc"
+	say_mod = "states" //inherited from a user's real species
+	sexes = 0
+	species_traits = list(NOTRANSSTING,NOEYESPRITES,NO_DNA_COPY,NOBLOOD,ROBOTIC_LIMBS,NOZOMBIE,MUTCOLORS,REVIVESBYHEALING,NOHUSK,NOMOUTH) //all of these + whatever we inherit from the real species
+	inherent_traits = list(TRAIT_RESISTCOLD,TRAIT_VIRUSIMMUNE,TRAIT_NOBREATH,TRAIT_RADIMMUNE,TRAIT_LIMBATTACHMENT)
+	inherent_biotypes = list(MOB_ROBOTIC, MOB_HUMANOID)
+	mutant_brain = /obj/item/organ/brain/positron
+	mutanteyes = /obj/item/organ/eyes/robotic
+	mutanttongue = /obj/item/organ/tongue/robot
+	mutantliver = /obj/item/organ/liver/cybernetic/upgraded/ipc
+	mutantstomach = /obj/item/organ/stomach/cell
+	mutantears = /obj/item/organ/ears/robot
+	mutant_organs = list(/obj/item/organ/cyberimp/arm/power_cord)
+	mutant_bodyparts = list("ipc_screen", "ipc_antenna", "ipc_chassis")
+	default_features = list("mcolor" = "#7D7D7D", "ipc_screen" = "Static", "ipc_antenna" = "None", "ipc_chassis" = "Morpheus Cyberkinetics(Greyscale)")
+	meat = /obj/item/stack/sheet/plasteel{amount = 5}
+	skinned_type = /obj/item/stack/sheet/iron{amount = 10}
+	exotic_blood = "oil"
+	damage_overlay_type = "synth"
+	limbs_id = "synth"
+	mutant_bodyparts = list("ipc_screen", "ipc_antenna", "ipc_chassis")
+	default_features = list("ipc_screen" = "BSOD", "ipc_antenna" = "None")
+	burnmod = 2
+	heatmod = 1.5
+	brutemod = 1
+	toxmod = 0
+	clonemod = 0
+	staminamod = 0.8
```

Comment:
That's a YIKES from me, dog. This, among other things, makes you take 3 stun baton strikes to fully take down.